Cargo.toml

```diff
@@ -49,6 +49,7 @@ cast_possible_wrap = "allow"
 cast_lossless = "allow"
 similar_names = "allow"
 implicit_hasher = "allow"
+unsafe_derive_deserialize = "allow"
```

Comment:
Not sure why it was giving this warning when deriving on types like `Coin`. Maybe the Python bindings are tripping it up